pubspec.yaml

```diff
@@ -23,6 +23,5 @@ dependencies:
   observable: '>=0.17.0+1 <0.23.0'
   protobuf: ^0.7.1
   perf_api: ^0.1.0
-  quiver: '>=0.24.0 <0.29.0'
-  sass_builder: ^1.1.2
-  uuid: ^0.5.3
+  quiver: '>=0.24.0 <0.30.0'
+  sass_builder: ^2.0.0
```

Comment:
Do we need a dependency override for the angular compiler?

Reply:
I haven't seen any problems. Can you elaborate on what you saw? Is it an issue with this version of the components?

I prefer to leave the version constraint up to the angular package unless something is broken. In the issue that was filed it looked like they were using mixed versions of angular and angular_components that probably were incompatible.

Reply:
@jakemac53 showed me the issue. It seems like there was a change in angular_compiler v0.4.0-alpha+12 that breaks if you use it with angular v5.0.0-alpha+11. Releasing this version should allow to version solve and upgrade to angular v5.0.0-alpha+12 which fixes the issue. Maybe we should ask angular to pin specific versions. I don't want to have to track the transitive dependencies of angular here.